src/atoms/sdp_cone/operatornorm.jl

```diff
@@ -42,6 +43,21 @@ end
 operatornorm(x::AbstractExpr) = OperatorNormAtom(x)
```

Comment:
Looks like these are exported; should `operatornorm(x)` be deprecated for `opnorm(x)`?

Reply:
Good idea. I've implemented that in a separate commit.